Documentation/Signoffs/sensor_subsystem.md

```diff
@@ -0,0 +1,149 @@
+| 3   | The sensor shall have a resolution great enough so that the golf ball spans 30 pixels from 6 feet       | Conceptual Design |
+| 4   | The sensor shall have a range of at least 6 feet                                                        | Conceptual Design |
+| 5   | The sensor shall have a FOV that is wider than 56" from 6' away                                         | Device Constraint |
+
+1. The sensor requires a USB connection for both data and power therefore also requiring a direct connection to the Jetson Nano processor
+2. The fastest time recorded from DEVCOM is 1.95 s for the golf ball to reach the bottom of its trajectory. Therefore, the sensor must be able to retrieve data for at least 2 positions so that speed can be calculated
+3. The sensor must have a resolution great enough so that the golf ball will span across 30 pixels for image processing to be able to discern where the golf ball is
+4. The furthest point that the sensor must be able to track and detect is the starting point of each golf ball which is about 6 feet from the launcher
+5. Since the width of anchor 2 from the DEVCOM rulebook is 56", the FOV must be wider than 56" at 6' away in order to encompass all the fishing lines
+
+## **Buildable Schematic**
+
+![Schematic](../Images/Sensor_subsystem/schematic.png)
+
+*Figure 1: Buildable Schematic of subsystem*
+
+## **Analysis:**
+
+### **Placement of Camera**
+
+According to Intel, the D435 has a minimum depth sensing range of about 28 cm or 11.02" [2]. Because of this, the camera should be mounted towards the back of the launcher so that the camera will be able to use the depth camera for the maximum amount of time. The camera should also be placed as high up within the 1' X 1' X 1' area for the launcher. Measuring from the 1/4-20 threaded mounting point on the bottom of the camera, it should be mounted 10" from the ground and 2" from the back of the launcher base. This mounting position allows for some room for the USB cable as well as the mounting bracket so that nothing exceeds the boundary of the launcher size constraint. In order to center the fishing lines in the FOV of the camera, the camera should be mounted at about a 21.64° angle from the horizontal. This angle was found using the angle of the fishing line for both variable heights. This angle points the center of the camera at a point directly 46.5" from the ground on the A-frame where anchor point 2 is. This height is in the middle of the 2 variable heights given in the DEVCOM rulebook. Using this height in combination with the camera height and distance from anchor point 2 this angle can be found. The distance from the back of the launcher to anchor 2 is shown in the image above to be 94".
+
+~~~ math
+
+arctan((46.5 - 10) / (94 - 2)) = 21.64°
+
+~~~
+
+Using this angle will allow every line to be visible within the 42° vertical FOV of the RGB camera
+
+### **Resolution**
```

Comment:
You have to explain what the math means with comments. Otherwise this is indecipherable.